Description: Add autogenerated upstream changelog
 Built from git commit lists between current and
 previous upstream tags.
Author: Philippe Thierry
Last-Update: 2017-06-01
--- /dev/null
+++ b/changelog
@@ -0,0 +1,11 @@
+# Release notes for version 3.4.2
+2017-01-11 13:20:51 -0800: [secretsquirrel] fix entry point instruction trucation (rare) and copyright updates
+2016-08-17 13:28:04 -0400: [Josh Pitts] Better CFG enum
+2016-08-16 22:25:50 -0400: [Josh Pitts] Update pebin.py
+2016-07-21 08:04:26 -0400: [Josh Pitts] Update README.md
+2016-07-21 08:04:17 -0400: [Josh Pitts] Update README.md
+2016-07-21 08:03:57 -0400: [Josh Pitts] Update README.md
+2016-07-21 08:02:41 -0400: [Josh Pitts] Merge pull request #69 from gitter-badger/gitter-badge-1
+2016-07-21 11:59:05 +0000: [The Gitter Badger] Add Gitter badge
+2016-07-05 11:37:01 -0400: [secret squirrel] Update README.md
+2016-07-05 09:45:41 -0400: [secret squirrel] Update README.md
